Source/Fuse.Controls.Navigation/NavigationControl.Pages.uno

```diff
@@ -70,12 +71,11 @@ namespace Fuse.Controls
 				
 			return path;
 		}
-		
+
+		[Uno.Obsolete]
 		protected void UpdateContextData( Visual page, object data )
```

Comment:
This should be mentioned in changelog, I think...

Reply:
Was just a note for me. I'll remove the function if I can (it wasn't released yet).

Reply:
Sounds good!